inlang/source-code/website/src/interface/sdkDocs/SdkDocsLayout.tsx

```diff
@@ -6,8 +6,8 @@ const SdkDocsLayout = (props: { children: JSXElement }) => {
 	return (
 		<div class="bg-surface-50 min-h-screen">
 			<SdkDocsHeader />
-			<div class="w-full px-4 min-h-[calc(100vh_-_107px_-_480px)]">
-				<div class="max-w-5xl mx-auto">{props.children}</div>
+			<div class="w-full px-4 md:px-8 xl:px-4 min-h-[calc(100vh_-_107px_-_480px)]">
+				<div class="max-w-7xl mx-auto">{props.children}</div>
 			</div>
 			<Footer />
```

Comment:
Maybe we can change the categories in the footer when visiting the SDK docs to the ones on the tab (SDK and plugins right now), if that's a quick change. Would feel more like a separated page then imo.

Reply:
Replaced Category with Documentation